service/csv/pokemon.go

```diff
@@ -0,0 +1,60 @@
+package service_csv
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/raymundo/academy-go-q42021/model"
+)
+
+func GetPokemons() []model.Pokemon {
+	f, err := os.Open("assets/pokemons.csv")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer f.Close()
+
+	csvReader := csv.NewReader(f)
+	data, err := csvReader.ReadAll()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	pokemons := formatPokemonList(data)
+	return pokemons
+}
+
+func formatPokemonList(data [][]string) []model.Pokemon {
+	var pokemons []model.Pokemon
+
+	for i, pokemon := range data {
+		var dataPokemon []string = pokemon
```

Comment:
Just curious, what's the purpose of defining the new variable `pokemonData`? Why don't you use `pokemon`?